README.md

```diff
@@ -31,7 +30,7 @@ There are two administration fields:
      * The XPath to the DOI element e.g. /mods:mods/mods:identifier[@type="doi"] 
      * A default is included and should serve most repositories, but you can change it if yours is located elsewhere.
 * Content models
-     * Choose which CModels are able to display badges (Currently only applies to Altmetrics)
```

Comment:
@bondjimbond I just checked the code and it appears that while the use of configured Content Models have been expanded it is **not** used for all modules. This could be remedied with a PR, but currently they are used in Altmetrics, Crossref and OADoi.
https://github.com/Islandora/islandora_badges/search?utf8=%E2%9C%93&q=islandora_badges_show_for_cmodel&type=

Reply:
https://jira.duraspace.org/browse/ISLANDORA-2092

Reply:
Ah, good catch - I forgot to check whether WoS and Scopus were covered. Will do some testing and maybe fix the wording here.

Also noticed that Crossref is not captured in the readme. Will fix.